doc/tutorials/remote-ssh-config.md

```diff
@@ -0,0 +1,123 @@
+Troubleshooting Connection Issues to Remote SSH Servers
+=======================================================
+
+EasyDeploy requires working SSH connections to remote servers in order to deploy
+and roll back the applications. This article summarizes the most common connection
+issues and proposes some solutions.
+
+Connection Refused
+------------------
+
+### There may be too many simultaneous SSH connections to the server
+
+Check the `MaxSessions` and `MaxStartups` config options of the SSH server.
+
+```
+# /etc/ssh/sshd_config
+# ...
+
+# Defines the maximum number of concurrent unauthenticated connections. Additional
+# connections are dropped until authentication succeeds. Default: 10:30:100
+# (if there are '10' connections, refuse '30'% of them and increase the drop rate
+# linearly until '100' connections are reached and then all are refused)
+MaxStartups 10:30:100
+
+# Defines the maximum number of open shell, login or subsystem (e.g. sftp)
+# sessions permitted per connection. This mostly affects to users with
+# multiplexing connections.
+MaxSessions 10
+```
+
+### The firewall of the remote server may be dropping your SSH connections
+
+Deploying an application requires making lots of SSH connections in a short
+period of time. Some firewalls may consider that a suspicious behavior and start
+dropping some of your SSH connections.
+
+### Remote servers may have disabled connections using passwords
```

Comment:
Are you sure that you will then run into a "Connection Refused" error? If I am not mistaken, SSH will tell you that you something like `Permission denied (publickey).`, won't it?